jwst/datamodels/schemas/dark.schema.yaml

```diff
@@ -1,25 +1,30 @@
 title: Dark current data model
 allOf:
 - $ref: referencefile.schema.yaml
+- $ref: instrument_name.schema.yaml
+- $ref: detector.schema.yaml
+- $ref: filter_core.schema.yaml
```

Comment:
That's a possibility. The existing "filter" schema is for a set of reference files that contain filter throughput curves, which are used in the NIRISS AMI level-3 processing. We could potentially rename that schema to something like "throughput" or "filter_throughput", so that your schema for the filter keyword could be just "filter".

Alternatively, what do folks think about naming the mini-schemas for individual keywords with something like a "_keyword" suffix, such as "filter_keyword.schema.yaml"? It requires more typing, but at the same time it does make it obvious that these schemas are dedicated to just defining a keyword and not an entire data model.